### PR TITLE
PS-9007 Change the link to download the AppArmor profile (8.1)

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
modified:   docs/apparmor.md

### CODE REVIEW

**Review by patrickbirch — APPROVED** [review]
Good

**Review by percona-ysorokin — APPROVED** [review]
LGTM

File: docs/apparmor.md [hunk 1]
```diff
@@ -33,6 +33,65 @@ $ sudo apt install apparmor-utils
...
 
+## Add the mysqld profile
+
+Add the mysqld profile with the following procedure:
+{.power-number}
+
+1. Download the current version of the AppArmor:
+
+    ```{.bash data-prompt="$"}
+    $ wget https://raw.githubusercontent.com/percona/percona-server/release-{{release}}/packaging/deb-in/extra/apparmor-profile
```
Comment (kamil-holubicki) [inline docs/apparmor.md]: This link is wrong. We've discussed it on Slack :)

Comment (alina-derkach-oaza) [reply docs/apparmor.md]: fixed

**Pull request merged**